packages/ssx-sdk/src/core.ts

```diff
@@ -162,21 +162,27 @@ export class SSXConnected implements ISSXConnected {
    * @returns Promise with nonce.
    */
   public async ssxServerNonce(params: Record<string, any>): Promise<string> {
+    const route = this.config.providers?.server?.routes?.nonce ?? '/ssx-nonce';
+    const requestConfig = isSSXRouteConfig(route)
+      ? route
+      : {
+          url: route,
+          customOperation: undefined,
+        };
+
+    const { customOperation } = requestConfig;
```

Comment:
Added the `customOperation` property. Allows a user to call a custom operation instead of the regular ssx operation for any endpoint.